### DISCUSSION TITLE
How to Duplicate a User/Profile

### DISCUSSION THREAD

**Discussion by Antonimo-Dev** [freeplane/freeplane#999] [Help] [answered]
I use multiple "profiles" to organise groups of maps and to run separate instances with these different groups.

I have a folder called "mm-01" containing a folder called 1.10x  as well as a bunch of *.mm mindmaps. I launch Freeplane using the following shortcut in Windows 11:

`"C:\Program Files\Freeplane\freeplane.exe" -U C:\Users\XXX\OneDrive\mm-01`

I duplicated the user profile to another folder called "mm-02".

When I start Freeplane with the command `"C:\Program Files\Freeplane\freeplane.exe" -U C:\Users\XXX\OneDrive\mm-02` I have lost some of the settings, including all of the assigned hotkeys.

Can anyone tell me where the assigned hotkeys are stored please?

Thanks in advance.

**Comment by dpolivaev** [comment, marked as answer]
accelerator.properties in `C:\Users\XXX\OneDrive\mm-01\<version>`

**Comment by Antonimo-Dev** [comment]
Hi Dimitry,

Thank you for your help. I copied the `<version>` folder over, but that did not seem to work properly, so I ran the command to launch Freeplane with the -u flag pointing to a non-existent `<version>` folder. The `<version>` folder was created automatically, then I was able to copy the entries from within **accelerator.properties** and **auto.properties** into the newly created files and everything seems to be good now.
